src/DateRange.vue

```diff
@@ -14,49 +19,56 @@
         </div>
         <div class="date-range__pickers">
           <div class="date-range__picker date-range__pickers--start">
-            <v-text-field name="startDate" :label="`Start Date(${format})`" class="date-range__pickers-input" prepend-icon="event" v-model="formattedStartDate"
-              readonly></v-text-field>
-            <v-date-picker @change="onDateRangeChange" :allowed-dates="allowedStartDates" no-title v-model="startDate"></v-date-picker>
+            <v-text-field
+              name="startDate"
+              :label="`Start Date(${format})`"
+              class="date-range__pickers-input"
+              prepend-icon="event"
+              v-model="formattedStartDate"
+              readonly/>
+            <v-date-picker
+              @change="onDateRangeChange"
+              :allowed-dates="allowedStartDates"
+              no-title
+              v-model="startDate"/>
           </div>
           <div class="date-range__picker date-range__picker--end">
-            <v-text-field name="endDate" :label="`End Date(${format})`" v-model="formattedEndDate" class="date-range__pickers-input"
-              readonly></v-text-field>
-            <v-date-picker @change="onDateRangeChange" :allowed-dates="allowedEndDates" no-title v-model="endDate"></v-date-picker>
+            <v-text-field
+              name="endDate"
+              :label="`End Date(${format})`"
+              v-model="formattedEndDate"
+              class="date-range__pickers-input"
+              readonly/>
+            <v-date-picker
+              @change="onDateRangeChange"
+              :allowed-dates="allowedEndDates"
+              no-title
+              v-model="endDate"/>
           </div>
         </div>
       </div>
     </v-card-text>
   </v-card>
 </template>
+
 <script>
-import * as moment from 'moment';
+import moment from 'moment';
+
 export default {
+  props: {
+    options: {
```

Comment:
@vinaypuppal thanks for doing this. I was thinking of doing this next.

Reply:
eslint was complaining before so I added validation for props